src/views/Home.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import Slider from "react-slick";
+import sliderData from '../mocks/en-us/featured-banners.json';
+import productCategoriesData from '../mocks/en-us/product-categories.json';
+import productsData from '../mocks/en-us/featured-products.json';
+import { sliderBannersSettings, sliderProductsSettings } from '../utils/constants.js';
+import Product from '../components/Product.js';
+
+class Home extends Component {
```

Comment:
Igual lo de la clase en este componente, creo que seria mejor cambiarlo a un functional component para llevar el proyecto por el mismo camino

Reply:
Changed to functional component.